src/js/game.ts

```diff
@@ -1185,6 +1211,9 @@ export class Game {
         ui.showItemFoundView();
         ui.setSleuthSpeakerText(ui.sleuthSpeakingFoundItMsg, true);
         this.speak(ui.sleuthSpeakingFoundItMsgEmojiName);
+        if (isIOS()) {
```

Comment:
Not sure necessary but maybe a little more defensive to implement this with an if else around the `this.speak(ui.sleuthSpeakingFoundItMsgEmojiName);` line above it so that we use one or the other for other platforms. Also if iOS starts working at some point we will have duplicate speaking calls with this.

Reply:
Here and elsewhere for `spriteSpeak ` calls